aws-globalaccelerator-accelerator/src/main/kotlin/software/amazon/globalaccelerator/accelerator/CreateHandler.kt

```diff
@@ -58,6 +58,6 @@ class CreateHandler : BaseHandler<CallbackContext?>() {
                 .withIpAddresses(model.ipAddresses)
                 .withTags(convertedTags)
                 .withIdempotencyToken(handlerRequest.clientRequestToken)
-        return proxy.injectCredentialsAndInvoke(request, { createAcceleratorRequest: CreateAcceleratorRequest? -> agaClient.createAccelerator(createAcceleratorRequest) }).accelerator
+        return proxy.injectCredentialsAndInvoke(request, agaClient::createAccelerator).accelerator
```

Comment:
I believe we will need to update listener and endpoint group resources as well. Can you add those to PR as well?

Reply:
The listener and endpoint group projects build successfully without changes and do not contain the same issue as the accelerator project